deployments/ugr01/hubploy.yaml

```diff
@@ -0,0 +1,17 @@
+images:
+  images:
+    - name: us-central1-docker.pkg.dev/ucb-datahub-2018/user-images/primary-user-image
```

Comment:
i think it will serve us better if we have a separate image for this hub.  i'd consider cribbing the image config from data100 and using that instead.